src/scss/tools/mixins/_map_deep_get.scss

```diff
@@ -0,0 +1,12 @@
+/// Map deep get
+/// @author Hugo Giraudel
+/// @access public
+/// @param {Map} $map - Map
+/// @param {Arglist} $keys - Key chain
+/// @return {*} - Desired value
```

Comment:
@GeorgeJegon this comment and description of mixin is in the format of documentjs?